pkg/lib/verifier/rules/pcr_eventlog_includes.go

```diff
@@ -5,66 +5,128 @@
 package rules
 
 import (
-	"github.com/intel-secl/intel-secl/v3/pkg/hvs/constants/verifier-rules-and-faults"
+	"fmt"
+
+	constants "github.com/intel-secl/intel-secl/v3/pkg/hvs/constants/verifier-rules-and-faults"
 	"github.com/intel-secl/intel-secl/v3/pkg/lib/flavor/common"
 	"github.com/intel-secl/intel-secl/v3/pkg/lib/host-connector/types"
 	"github.com/intel-secl/intel-secl/v3/pkg/model/hvs"
 	"github.com/pkg/errors"
 )
 
-func NewPcrEventLogIncludes(expectedEventLogEntry *types.EventLogEntry, expectedPcr *types.Pcr, marker common.FlavorPart) (Rule, error) {
-	if expectedEventLogEntry == nil {
+//NewPcrEventLogIncludes creates the rule that will check
+//if all the actual event log measurements included in expected
+func NewPcrEventLogIncludes(expectedEventLogEntry *types.EventLogEntry, expectedPcrEventLogEntry *types.TpmEventLog, expectedPcr *types.Pcr, marker common.FlavorPart) (Rule, error) {
```

Comment:
Again, why two parameters for event logs?

Reply:
Here the two logs parameters are used for both the Linux and exsi flows as now both are having different structures (old and new implementation). When we start implementing dynamic verification for exsi flow, we can keep it as a single expected entry as these flows will have the same structure.